### PR TITLE
docs: update coc

### PR DESCRIPTION
**Pull request by dhmlau**
Signed-off-by: Diana Lau <dhmlau@ca.ibm.com>

Update the COC to  include OpenJS Foundation escalation path. 

cc @achrinza 

## Checklist

- [x] DCO (Developer Certificate of Origin) [signed in all commits](https://loopback.io/doc/en/contrib/code-contrib.html)
- [ ] `npm test` passes on your machine
- [ ] New tests added or existing tests modified to cover all changes
- [ ] Code conforms with the [style guide](http://loopback.io/doc/en/contrib/style-guide.html)
- [ ] API Documentation in code was updated
- [ ] Documentation in [/docs/site](../tree/master/docs/site) was updated
- [ ] Affected artifact templates in `packages/cli` were updated
- [ ] Affected example projects in `examples/*` were updated

👉 [Check out how to submit a PR](https://loopback.io/doc/en/lb4/submitting_a_pr.html) 👈

### CODE REVIEW

File: docs/CODE_OF_CONDUCT.md [hunk 1]
```diff
@@ -1,4 +1,54 @@
...
+- [github.com/ContributorCovenant](https://github.com/ContributorCovenant/contributor_covenant/tree/release/content/version/2/0)
+
+Refer to the sections on reporting and escalation in this document for the
+specific emails that can be used to report and escalate issues.
+
+## Reporting
+
+### Project Spaces
+
+For reporting issues in spaces related to LoopBack please use the email
```
Comment (achrinza) [inline docs/CODE_OF_CONDUCT.md:19]:
```suggestion
For reporting issues in spaces related to LoopBack, please use the email
```

File: docs/CODE_OF_CONDUCT.md [hunk 2]
```diff
@@ -1,4 +1,54 @@
...
+`tsc@loopback.io`. LoopBack handles CoC issues related to the spaces that it
+maintains. Projects maintainers commit to:
```
Comment (achrinza) [inline docs/CODE_OF_CONDUCT.md:21]: Ditto, not all maintainers are in the TSC.

```suggestion
maintains. The project TSC commit to:
```

File: docs/CODE_OF_CONDUCT.md [hunk 3]
```diff
@@ -1,4 +1,54 @@
...
+
+Refer to the sections on reporting and escalation in this document for the
+specific emails that can be used to report and escalate issues.
+
+## Reporting
+
+### Project Spaces
+
+For reporting issues in spaces related to LoopBack please use the email
+`tsc@loopback.io`. LoopBack handles CoC issues related to the spaces that it
```
Comment (achrinza) [inline docs/CODE_OF_CONDUCT.md:20]: Perhaps we want to be specific that it's the TSC?

```suggestion
`tsc@loopback.io`. The LoopBack Technical Steering Committee (TSC) handles CoC issues related to the spaces that it
```

**Review by achrinza — CHANGES_REQUESTED** [review]

**Comment by dhmlau** [comment]
@achrinza, i've incorporated your changes. PTAL again. Thanks.

File: docs/CODE_OF_CONDUCT.md [hunk 4]
```diff
@@ -1,4 +1,54 @@
...
+Refer to the sections on reporting and escalation in this document for the
+specific emails that can be used to report and escalate issues.
+
+## Reporting
+
+### Project Spaces
+
+For reporting issues in spaces related to LoopBack, please use the email
+`tsc@loopback.io`. The LoopBack Technical Steering Committee (TSC) handles CoC issues related to the spaces that it
+maintains. The project TSC commit to:
```
Comment (achrinza) [inline docs/CODE_OF_CONDUCT.md:21]: Nitpick: grammar
```suggestion
maintains. The project TSC commits to:
```

Sorry I didn't catch it the first time :bowing_man:

**Review by achrinza — CHANGES_REQUESTED** [review]

**Review by achrinza — APPROVED** [review]

**Pull request merged**